srv/admin-service.cds

```diff
@@ -1,28 +1,45 @@
-using { sap.common.Languages as CommonLanguages } from '@sap/cds/common';
-using { my.bookshop as my } from '../db/schema';
+using
+{sap.common.Languages as CommonLanguages} from '@sap/cds/common';
+using
+{my.bookshop as my} from '../db/index';
 
-@path:'admin'
-service AdminService @(requires:'admin') {
-  entity Books as projection on my.Books actions {
-    action addToOrder(order_ID: UUID, amount: Integer) returns Orders;
+@path : 'admin'
+service AdminService @(requires : 'admin')
+{
+  entity Books   as projection on my.Books actions
+  {
+    action addToOrder(order_ID : UUID, amount : Integer) returns Orders;
   }
+
   entity Authors as projection on my.Authors;
-  entity Orders as select from my.Orders;
+  entity Orders  as select from my.Orders;
 }
 
 // Deep Search Items
-annotate AdminService.Orders with @cds.search : {OrderNo, Items};
-annotate AdminService.OrderItems with @cds.search : {book};
-annotate AdminService.Books with @cds.search : {descr, title};
+annotate AdminService.Orders with @cds.search :
+{
+  OrderNo,
+  Items
+};
+
+annotate AdminService.OrderItems with @cds.search :
+{book};
+
+annotate AdminService.Books with @cds.search :
+{
+  descr,
+  title
+};
 
 // Enable Fiori Draft for Orders
 annotate AdminService.Orders with @odata.draft.enabled;
-annotate AdminService.Books with @odata.draft.enabled @Capabilities.InsertRestrictions.Insertable: false;
+annotate AdminService.Books with @odata.draft.enabled;
```

Comment:
This annotation prevents to add new books to the bookshop which I think the admin should be able to.

Reply:
We added this intentionally to show that draft also works with update only entities. But yes from a business perspective this might not be the most reasonable thing..